packages/layers/system-x/package.mask/10-mocaccino-x.mask

```diff
@@ -0,0 +1,2 @@
+dev-lang/python:3.9
```

Comment:
Then we can remove the `emerge --unmerge`... right?

Reply:
:thinking: , I'm not sure of this because this is the version that arrives from stage3 or it will arrive for sure in the next stage3, IIRC.

Reply:
I think that maintain it, it's a safe way to ensure we drop always the python version not supported by mocaccino. The mask instead it's used to avoid injection on rebuild.